docker-compose.yml

```diff
@@ -411,11 +416,6 @@ services:
       - 'scanstore_rescue:${SCANSTORE_RESCUE_HARVESTING_ENDPOINT_REMOTE}'
       - 'kaplan_rescue:${KAPLAN_RESCUE_HARVESTING_ENDPOINT_REMOTE}'
       - 'colenda_workspace:${REMOTE_WORKSPACE}'
-  rabbitmq:
```

Comment:
Could we remove RabbitMq in a separate PR? I don't think its used in any significant way, but the app does send messages to it so I would have to make some other changes to the code.

Reply:
This is actually duplicated code and doesn't actually remove RabbitMQ. The service still exists: https://github.com/upenn-libraries/bulwark/blob/3c051fc7d847f2029c9fc3ac2c8166a1832c029e/docker-compose.yml#L336-L340

Reply:
oh, I see! Sorry for the confusion!